### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HugoLazaro**

### CODE REVIEW

**Comment by fjlopez** [comment]
Check your email inbox for an update to the lab send through Moodle. It is a fix to ensure that the badge shows that the CI has been triggered by a push request to the work branch.

**Comment by HugoLazaro** [comment]
I have changed the build status.

**Comment by fjlopez** [comment]
**Push** a minimum change to the **work** branch (e.g. a file with additional whitespace) to trigger the action.

**Comment by fjlopez** [comment]
Do it in **work**

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -2,9 +2,6 @@ name: CI
 on:
   push:
     branches: [ work ]
-  pull_request:
```
Comment (fjlopez) [inline .github/workflows/CI.yml:5]: This change should be done in the **work** branch.

**Review by fjlopez — CHANGES_REQUESTED** [review]
This change should be done in

**Comment by fjlopez** [comment]
Remember, you need to make a **push** to your **work** branch (e.g add a whitespace to a file) to trigger the build.

**Comment by fjlopez** [comment]
You don't need to close it

**Comment by fjlopez** [comment]
Just make the push and the action will run. You can warn me here that the action has been successful.

**Comment by HugoLazaro** [comment]
I didnt want to close the pull request, I have pushed to work

**Pull request merged**